CHANGELOG.unreleased.md

```diff
@@ -52,9 +53,12 @@
 - @xen-orchestra/xapi major
 - @vates/read-chunk minor
 - complex-matcher patch
+  <<<<<<< HEAD
 - xen-api patch
 - xo-server patch
-- xo-server-transport-xmpp patch
+- # xo-server-transport-xmpp patch
+- xo-server minor
+  > > > > > > > ca3d460f1... changelog.md
```

Comment:
Incorrect.

Reply:
Hm, I don't understand what my IDE did. BTW it's fixed now.

Reply:
Of course, it's your IDE's fault :grin: